modular_zubbers/code/modules/antagonists/bloodsucker/bloodsuckers/integration.dm

```diff
@@ -5,10 +5,12 @@
 	return ..()
 
 /datum/reagent/blood/expose_mob(mob/living/exposed_mob, methods=TOUCH, reac_volume, show_message=TRUE, touch_protection=0)
-	var/datum/antagonist/bloodsucker/bloodsuckerdatum = IS_BLOODSUCKER(exposed_mob)
-	if(!bloodsuckerdatum)
-		return ..()
-	bloodsuckerdatum.bloodsucker_blood_volume = min(bloodsuckerdatum.bloodsucker_blood_volume + round(reac_volume, 0.1), BLOOD_VOLUME_MAXIMUM)
+    var/datum/antagonist/bloodsucker/bloodsuckerdatum = IS_BLOODSUCKER(exposed_mob)
+    if(!bloodsuckerdatum)
+        return ..()
+    var/blood_to_add = min(bloodsuckerdatum.bloodsucker_blood_volume + round(reac_volume, 0.1), BLOOD_VOLUME_NORMAL)
+    if(blood_to_add > 0)
```

Comment:
actually sorry for this if statement you should check if it's above zero after blood_to_add - bloodsuckerdatum.bloodsucker_blood_volume

Reply:
You sent me the correction over discord and i've committed that. Ergo resolving.